### ISSUE TITLE
Error appears when clicking flight master 

### ISSUE THREAD

**Issue by Cornelis83** [Aviana/LunaUnitFrames#628]
Every first time clicking a flight master after logging into wow, I get this error:

Date: 2019-12-23 12:21:21
ID: 1
Error occured in: AddOn: LunaUnitFrames
Count: 1
Message: Note: AddOn LunaUnitFrames attempted to call a protected function (LUFHeaderpartypetUnitButton1:SetAttribute()) during combat lockdown.
Debug:
   [C]: SetAttribute()
   LunaUnitFrames\modules\units.lua:393:
      LunaUnitFrames\modules\units.lua:351
   [C]: ?
   [C]: ?
   ..\FrameXML\SecureHandlers.lua:266:
      ..\FrameXML\SecureHandlers.lua:263
   [C]: ?
   ..\FrameXML\SecureHandlers.lua:430:
      ..\FrameXML\SecureHandlers.lua:417
   (tail call): ?
   [C]: SetAttribute()
   ..\FrameXML\RestrictedFrames.lua:481: SetAttribute()
   [string "			if( name == "unit" ) then..."]:6:
      [string "			if( name == "unit" ) then..."]:1
   (tail call): ?
   [C]: ?
   ..\FrameXML\RestrictedExecution.lua:484:
      ..\FrameXML\RestrictedExecution.lua:447
   (tail call): ?
   ..\FrameXML\SecureHandlers.lua:422:
      ..\FrameXML\SecureHandlers.lua:417
   (tail call): ?
   [C]: SetAttribute()
   ..\FrameXML\SecureGroupHeaders.lua:203:
      ..\FrameXML\SecureGroupHeaders.lua:123
   ..\FrameXML\SecureGroupHeaders.lua:488: SecureGroupHeader_Update()
   ..\FrameXML\SecureGroupHeaders.lua:62:
      ..\FrameXML\SecureGroupHeaders.lua:60
Locals:
None
AddOns:
  Swatter, v8.2.6377 (SwimmingSeadragon)
  AdvancedInterfaceOptions, v1.3.3
  Atlas, vv1.45.01.11302
  AtlasClassicWoW, vv1.45.00.11302.05
  AtlasLootClassic, vv1.4.3
  AtlasLootClassicData, vv1.4.3
  AtlasLootClassicDungeonsAndRaids, vv1.4.3
  AucAdvanced, v8.2.6430 (SwimmingSeadragon)
  auccheapsearcher, v20191013-retail
  AucFilterBasic, v8.2.6364 (SwimmingSeadragon)
  AucScanData, v8.2.6365 (SwimmingSeadragon)
  AucStatHistogram, v8.2.6366 (SwimmingSeadragon)
  AucStatiLevel, v8.2.6370 (SwimmingSeadragon)
  AucStatPurchased, v8.2.6367 (SwimmingSeadragon)
  AucStatSimple, v8.2.6399 (SwimmingSeadragon)
  AucStatStdDev, v8.2.6369 (SwimmingSeadragon)
  AucUtilFixAH, v8.2.6371 (SwimmingSeadragon)
  Auctionator, v8.1.0
  Bartender4, v4.8.9
  BeanCounter, v8.2.6434 (SwimmingSeadragon)
  BetterVendorPrice, vv1.11.00
  ClassicThreatMeter, v1.10
  DBMCore, v1.13.25
  DBMDefaultSkin, v
  DBMStatusBarTimers, v
  DejaClassicStats, v1303r001
  Details, v
  DetailsEncounterDetails, v
  DetailsRaidCheck, v
  DetailsStreamer, v
  DetailsTinyThreat, v
  DetailsVanguard, v
  Enchantrix, v8.2.6428 (SwimmingSeadragon)
  EnchantrixBarker, v8.2.6469 (SwimmingSeadragon)
  InFlightLoad, v1.13.003
  Informant, v8.2.6374 (SwimmingSeadragon)
  LeatrixMaps, v1.13.40
  LeatrixPlus, v1.13.41
  LetMeCast, v1.3
  LunaUnitFrames, v1.0
  MikScrollingBattleText, v5.7.149
  ModernQuestWatch, v1.0.9
  OmniCC, v8.3.1
  Prat30, v3.7.45
  Prat30Libraries, v
  Questie, v5.4.0
  RealMobHealth, v2.17
  SlideBar, v8.2.6375 (SwimmingSeadragon)
  Stubby, v8.2.6376 (SwimmingSeadragon)
  TidyPlatesThreatPlates, v1.2.6
  TitanClassic, v1.1.5.11303
  TitanClassicAmmo, v1.1.5.11303
  TitanClassicBag, v1.1.5.11303
  TitanClassicClock, v1.1.5.11303
  TitanClassicGold, v1.1.5.11303
  TitanClassicLocation, v1.1.5.11303
  TitanClassicLootType, v1.1.5.11303
  TitanClassicPerformance, v1.1.5.11303
  TitanClassicRegen, v1.1.5.11303
  TitanClassicRepair, v1.1.5.11303
  TitanClassicVolume, v1.1.5.11303
  TitanClassicXP, v1.1.5.11303
  tullaRange, v8.2.6
  XandY, v2.13
  BlizRuntimeLib_enUS v1.13.3.11303 <none>
  (ck=82a)

It states it appears in this add on so I guessed it was best posting here.

**Comment by Aviana** [comment]
Pretty sure you are running an outdated version.

**Comment by Cornelis83** [comment]
Could be, but I did a full update of all my addons via the twitchtv tool. I do that daily before I start up WoW as there were more issues with other addons since BG's were released. Most of them are fixed by now but only the error I mentioned above keeps popping up. So if what you say is true it means twitchtv doesn't offer me the newest version. And if that is true, where can I get the newest version.

While typing this message, I started to check the version of the addon in my AddOns map and I see a lot of dates in august. So I start to believe that I made the assumption the addon was always updated by twitchtv tool but I see now it isn't even listed there. I surely got it from a different source I guess...
